lib/src/native/realm_core.dart

```diff
@@ -725,6 +729,10 @@ class RealmObjectChangesHandle extends Handle<realm_object_changes> {
   RealmObjectChangesHandle._(Pointer<realm_object_changes> pointer) : super(pointer, 256);
 }
 
+class RealmAppCredentialsHandle extends Handle<realm_app_credentials> {
+  RealmAppCredentialsHandle._(Pointer<realm_app_credentials> pointer) : super(pointer, 256); // TODO: What should hint be?
```

Comment:
Can we set there the correct size

Reply:
Fixed in https://github.com/realm/realm-dart/pull/443/commits/7784c23f08bbc22471fe403d5a4a1bf0af6e94f5